index.js

```diff
@@ -6,14 +6,7 @@ module.exports = {
   included() {
     this._super.included.apply(this, arguments);
 
-    const resolve = require('resolve');
```

Comment:
We can remove the `resolve` package now.

Reply:
Done